### PR TITLE
some fixes

### PR DESCRIPTION
**Pull request by Scraayp**
v 2.5.0 update pr!

### CODE REVIEW

**Comment by Scraayp** [comment]
Update v2.5.0
- Added mute
- Added modlogs
- Added removemessages
- Fixed bugs!

Has been coded. Only checks by the maintainer needed! Everything has been tested and works well!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<p align="center"><a href="https://nodei.co/npm/discord-anti-spam/"><img src="https://nodei.co/npm/discord-anti-spam.png"></a></p>
+# THIS IS THE DEV BRANCH! It's been used for development, DO NOT USE THIS OPTIONS FOR MAIN USE!!!!!!!
```
Comment (Androz2091) [inline README.md:1]:
```suggestion
<p align="center"><a href="https://nodei.co/npm/discord-anti-spam/"><img src="https://nodei.co/npm/discord-anti-spam.png"></a></p>
```

You should not change this text, it will complicate the merge into the master branch once we will want to release the updae

File: index.d.ts [hunk 2]
```diff
@@ -81,5 +81,8 @@ declare module 'discord-anti-spam' {
 		kickEnabled?: boolean;
 		banEnabled?: boolean;
 		muteEnabled?: boolean;
+		muteRoleName?: String;
```
Comment (Androz2091) [inline index.d.ts:84]: `muteRoleName` is already specified line 62, right?

File: index.d.ts [hunk 3]
```diff
@@ -81,5 +81,8 @@ declare module 'discord-anti-spam' {
...
+		modLogsChannelName?: String;
```
Comment (Androz2091) [inline index.d.ts:85]:
```suggestion
		modLogsChannelName?: string;
```

File: index.js [hunk 4]
```diff
@@ -311,7 +312,7 @@ class AntiSpamClient extends EventEmitter {
 		}
 		this.cache.messages = this.cache.messages.filter((u) => u.authorID !== message.author.id)
 		this.cache.mutedUsers.push(message.author.id)
-		const role = message.guild.roles.cache.find(role => role.name === this.options.muteRoleName)
+		const role = message.guild.roles.cache.find(role => role.name === "Muted")
```
Comment (Androz2091) [inline index.js:315]: Why did you change this? I think we should keep `this.options.muteRoleName` no?

**Review by Androz2091 — CHANGES_REQUESTED** [review]
Thank you for your PR 👍 
Here are some review comments. And I think that you should remove the changes you made to the package.json file, there are useless and sometimes totally incorrect :wink:

**Review by Androz2091 — APPROVED** [review]
This should be merged soon

File: package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "discord-anti-spam",
-  "version": "2.4.1",
+  "version": "2.5.0",
```
Comment (Androz2091) [inline package.json:3]:
```suggestion
  "version": "2.4.1",
```
Version should only be bumped on merge

**Comment by Scraayp** [comment]
Not yet,

Need to fix the docs...

**Comment by Androz2091** [comment]
All right, ask for a review when it's done

**Comment by Scraayp** [comment]
Can' t do it today, I hope I will have it done tommorow morning!

**Comment by Scraayp** [comment]
It' s done.....
Took me 1 hour.... Hope it will now make people read it!

**Review by ToledoSDL — APPROVED** [review]

**Review by Androz2091 — APPROVED** [review]

**Comment by Scraayp** [comment]
Now it can be merged i guess...
